avocado/core/sysinfo.py

```diff
@@ -457,10 +480,13 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
         self.start_test_collectibles = set()
         self.end_test_collectibles = set()
 
+        self.end_test_fail_collectibles = set()
```

Comment:
let's remove the common files present in `end_test_collectibles` and `end_test_fail_collectibles`, users are free to add same files and commands in both places, it is better to find only the additional newer ones compared to `end_test_collectibles` and add to `end_test_fail_collectibles`.

so we can just be sure only additional files and commands needed for failed cases and without breaking the flow of all status..

E:g:- expecting like below... 
>>> a={2,3,4} # end_test_collectibles
>>> b={2,5,6} # end_test_fail_collectibles
>>> b = b-a
>>> b
{5, 6}

Reply:
Sure, will remove duplicates from end_test_fail_collectibles

Reply:
Sure, will remove duplicates from end_test_fail_collectibles

Reply:
done it immediately after generating the list. please check.